README.md

```diff
@@ -31,83 +33,140 @@ The extension provides a fast and accurate solution for:
+9. **Vascular Segments Volume:**
+   - Use the dropdown menu to select the vascular segments volume. The available options depend on the pre-loaded volumes in the software.
+
+The resectogram can only be used after distance maps have been calculated and uploaded.
+The following GIF demonstrates the usage of the Resectogram section in Slicer-Liver:
+
+![Slicer-Liver_resectogram.gif](Screenshots_tutorial/Slicer-Liver_resectogram.gif)
 
 ### Liver Segments
 
-Our approach to liver vascular territory segments definition consist of the defintion of a vascular territory by the centerline connecting user-defined sets of points. These centerlines will be the base for computation of liver segments in image space. The computation is based on shortest-distance mapping.
-The Liver segments can be defined using the following steps:
-
-1.  Select `Vascular Territory Segmentation`
-2.  Select the `Segmentation`.
-3.  You can hide the liver segmentation for better visibility in the 3D view by first selecting the liver segmentation, then select `Hide`.
-4.  Select the hepatic/portal segmentation for `Segment`. The 3D view will then automatically switch to landmark "Place Mode" 
-and a `Vessel points` list will be automatically generated.
-5.  Place user-defined markup landmarks along the selected hepatic/portal model in 3D View to mark the part of the vessel where centerlines should be generated.
-6.  Once all the points are placed, Click `Add Vessel Centerline`.
-7.  Optionally, repeat steps 4-6 for generation of additional centerline for other vessel segments (portal/hepatic).
-8.  For generation of several vascular territory segments, select `Vascular Territory` and "Create new territory ID".
-9.  Repeat steps 4 to 7 for creating centerlines to be used for calculation of the new Vascular Territory.
-10.  Click on `Calculate Vascular Territory Segmentation`. The generated vascular territories will then be visible in the 3D- and 2D Views.
-11. Multiple sets of vascular territories can be generated and stored in separat Vascular Territory Segmentations.
-To generate a new set of vascular territories, select "Create New Vascular Territory Segmentation" in `Vascular Territory Segmentation`. 
-Then repeat steps 2-9.
-
-![Liver segments -- placing fiducials](https://github.com/ALive-research/Slicer-Liver/blob/master/Screenshots/Slicer-Liver_screenshot_06.png?raw=true)
-
-![Liver segments -- placing fiducials](https://github.com/ALive-research/Slicer-Liver/blob/master/Screenshots/Slicer-Liver_screenshot_08.png?raw=true)
-
- ![Slicer-Liver_screenshot_14](https://github.com/dalbenzioG/Slicer-Liver/blob/master/Screenshots/Slicer-Liver_screenshot_14.jpg?raw=true)
+Our method for liver segment classification involves defining segments using centerlines created from user-specified points. These centerlines serve as the foundation for computing liver segments within the image space. The computation leverages shortest-distance mapping
+
```

Comment:
I see you have simplified the description a bit, by removing the repeat steps, and text about multiple vascular territories. This is probably a good thing. We may however keep the text about how to create multiple vascular territories in a separate step at the end?